Car1/src/car1/CarSetup.java

```diff
@@ -0,0 +1,104 @@
+
+package car1;
+
+/**
+ *The CarSetup class contains methods that will be used in the Car1 program.
+ * @author mkavic
+ */
+public class CarSetup {
```

Comment:
The class should just be named Car per the assignment but I do understand where you were going here in terms of showing the chicken vs the egg.